src/bucket.ts

```diff
@@ -87,7 +87,7 @@ interface MetadataOptions {
   ifMetagenerationNotMatch?: number | string;
 }
 
-export type GetFilesResponse = [File[], {}, unknown];
+export type GetFilesResponse = [File[], (GetFilesOptions | GetFilesCallback) & Partial<Pick<GetFilesOptions, 'pageToken'>>, unknown];
```

Comment:
You can omit `GetFilesCallback`, `query` can only be `GetFilesOptions` or an empty object. It will never get assigned the callback.

Reply:
I fixed it.